docs/book/src/quick-start.md

```diff
@@ -57,6 +57,29 @@ cd $GOPATH/src/example
 kubebuilder init --domain my.domain
 ```
 
+If there is an error similar to `dial tcp x.x.x.x:443: i/o timeout`, you need to set go proxy.
+Go version >= 1.13
```

Comment:
It is in the pre-requirements section already.

Reply:
All right,It is more appropriate to the issue or FAQ